examples/panic/panic.description

```diff
@@ -1,8 +1,8 @@
-// In Ballerina, panic means something unexpected has occured
+// In Ballerina, panic indicates abnormal completions, and usually implies something unexpected has occured
 // (something that should not have occurred during the normal operation). A panic causes
 // the call stack to be unwound until a matching catcher is found.
```

Comment:
"to be unwound" or "to unwind"?

Reply:
and catcher or trapper?

Reply:
Maybe something like the following?
```suggestion
// the call stack to unwind until it is trapped.
```

Reply:
`Unwind` seems to be the more suitable term.